Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -672,9 +672,11 @@ public SelectGestureRecognizer() : base(Tapped)
 			static void Tapped(UIGestureRecognizer recognizer)
 			{
 				var selector = (SelectGestureRecognizer)recognizer;
-
 				var table = (UITableView)recognizer.View;
 
+				if (selector._lastPath == null)
```

Comment:
Should we move this above the UITableView cast? Seems like we could avoid doing that since it wont go any further.